app/controllers/spree/admin/users_controller_decorator.rb

```diff
@@ -1,6 +1,8 @@
 Spree::Admin::UsersController.class_eval do
   append_before_action :load_user_orders, only: :edit
 
+  def edit; end
```

Comment:
Do we need this empty edit method?

Reply:
@dgra Rubocop said I had to explicitly define edit on the controller, and I noticed on a file in Sigurd that they had done it this way, so I tried it and Rubocop passed ¯\_(ツ)_/¯

Reply:
I'm just concerned this could be overwriting the default method. Since `Spree::Admin::UsersController` doesn't define `edit` itself, we may be able to get away with using 
```
def edit
  super
end
```